src/transaction/log_recovery.c

```diff
@@ -63,6 +66,22 @@ static void log_rv_redo_record (THREAD_ENTRY * thread_p, log_reader & log_pgptr_
 				int (*redofun) (THREAD_ENTRY * thread_p, LOG_RCV *), LOG_RCV * rcv,
 				const LOG_LSA * rcv_lsa_ptr, int undo_length, const char *undo_data,
 				LOG_ZIP & redo_unzip);
+static bool
+log_rv_fix_page_and_check_redo_is_needed (THREAD_ENTRY * thread_p, const VPID & page_vpid, log_rcv & rcv,
+					  LOG_RCVINDEX rcvindex, const log_lsa & rcv_lsa, const LOG_LSA * end_redo_lsa);
+
+// *INDENT-OFF*
+template <typename T>
+static int log_rv_get_log_rec_redo_data (THREAD_ENTRY *thread_p, log_reader &log_pgptr_reader, const T &log_rec,
+    log_rcv &rcv, log_rectype log_rtype, struct log_zip &undo_unzip_support, struct log_zip &redo_unzip_support);
+template <typename T>
+static void log_rv_redo_record_debug_logging_pre_data_retrieve (const log_lsa &rcv_lsa, LOG_RCVINDEX rcvindex,
```

Comment:
Since there is no post logging, can the function name be shortened as log_rv_redo_record_debug_logging?

Reply:
done